addons/purchase_requisition/migrations/10.0.0.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,67 @@
+---Fields in module 'purchase_requisition'---
+purchase_requisition / purchase.order.line      / quantity_tendered (float)     : DEL
+purchase_requisition / purchase.requisition     / exclusive (selection)         : DEL required: required, selection_keys: ['exclusive', 'multiple'], req_default: multiple
+purchase_requisition / purchase.requisition     / multiple_rfq_per_supplier (boolean): DEL
+# Nothing to do
+
+purchase_requisition / purchase.requisition     / type_id (many2one)            : NEW relation: purchase.requisition.type, required: required, req_default: function
```

Comment:
As this is created before having data loaded, the default is not applied. You should reapply it after, and try to avoid the ERROR on NOT NULL constraint meanwhile.